.github/workflows/_charm-quality-checks.yaml

```diff
@@ -292,3 +292,24 @@ jobs:
         with:
           timeout-minutes: 30
           limit-access-to-actor: true
+
+      - name: Show logs
+        run: | 
+          if test -d ./.logs; then
+            for filename in ./.logs/*.txt; do
+                for ((i=0; i<=3; i++)); do
+                      echo "::group::{{ $filename }}"
+                      cat "$filename"
+                      echo "::endgroup::"
+                done
```

Comment:
I'm not sure I understand what this for loop is doing; are we printing each log file 3 times?